fiatconnect-api.md

```diff
@@ -2213,6 +2214,23 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+####  8.3.2.4. `BankAccount`
```

Comment:
How about renaming this to IBANNumber? Since this isn't the only fiat account schema that represents a bank account

Reply:
Yeah, I am ok with that

Reply:
```suggestion
####  8.3.2.4. `IBANNumber`
```